source/healthBar.js

```diff
@@ -58,4 +61,11 @@ export class HealthBar {
         return (wasDead);
     }
 
+    show() {
+        this.healthBar.visible = true;
+    }
+    
+    hide() {
+        this.healthBar.visible = false;
+    }
```

Comment:
Assuming the health bar won't need to be hidden separately, I would just trash these and inside the enemy hide function do this:
`show() {
        this.enemy.visible = true;
        this.healthBar.visible = true;
    }
`

Reply:
Aight

Reply:
Oh, this would actually come out as `this.healthBar.healthBar.visible = false;`, still prefer it?

Reply:
Yikes. Leave it how you had it lol